cmake/FindSndFile.cmake

```diff
@@ -1,43 +1,43 @@
-# - Try to find SndFile
-# Once done this will define
-#
-#  SNDFILE_FOUND - system has SndFile
-#  SNDFILE_INCLUDE_DIRS - the SndFile include directory
-#  SNDFILE_LIBRARIES - Link these to use SndFile
-#
-#  Copyright © 2006  Wengo
-#  Copyright © 2009 Guillaume Martres
-#
-#  Redistribution and use is allowed according to the terms of the New
-#  BSD license.
-#  For details see the accompanying COPYING-CMAKE-SCRIPTS file.
+# Variables defined:
```

Comment:
Where is this file from? Copyright?

Reply:
It's from libsamplerate repository, no copyright. https://github.com/libsndfile/libsamplerate/blob/master/cmake/FindSndFile.cmake

Reply:
Alright, could you leave that link there, please?